certified-connectors/Asite (Hong Kong)/apiProperties.json

```diff
@@ -0,0 +1,56 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "{{ client id }}",
+          "scopes": [
+            "DOCUMENT_READ DOCUMENT_WRITE"
+          ],
+          "redirectMode": "Global",
```

Comment:
Please modify redirectMode as "GlobalPerConnector" instead Global.

Reply:
@vmanoharas I have changed as per your suggestion.

Reply:
@asitedeveloper, still the redirectMode is "Global" please change as "redirectMode": "GlobalPerConnector"

Reply:
![image](https://github.com/microsoft/PowerPlatformConnectors/assets/77782776/faf8750b-db76-4b70-9d1d-6a942a61cfa5)

I have already changed "GlobalPerConnector" in this file but I don't how why it is not reflecting.
Please refer attached screenshot.
commit Id : 43b4415a16384e53b90c20df6b5f57500a73151d